src/beautifiers/r-beautifier/index.coffee

```diff
@@ -0,0 +1,24 @@
+###
+Requires [formatR](https://github.com/yihui/formatR)
+###
+path = require("path")
+
+"use strict"
+Beautifier = require('../beautifier')
+
+module.exports = class R extends Beautifier
+  name: "R beautifier"
+  link: "https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/r-beautifier/r-beautifier.r"
```

Comment:
Ok, I will change it. Is it possible to add two links? One for R and one for formatR?

module.exports = class R extends Beautifier
  name: "formatR"
  link: "https://github.com/yihui/formatR"
  link: "https://cran.r-project.org/"